### ISSUE TITLE
TypeError: expected string or bytes-like object

### ISSUE THREAD

**Issue by vashilov** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#159]
Error:
```
File "/config/custom_components/xiaomi_cloud_map_extractor/common/image_handler.py", line 223, in draw_func
    font = ImageFont.truetype(font_file, font_size)
  File "/usr/local/lib/python3.9/site-packages/PIL/ImageFont.py", line 853, in truetype
    return freetype(font)
  File "/usr/local/lib/python3.9/site-packages/PIL/ImageFont.py", line 850, in freetype
    return FreeTypeFont(font, size, index, encoding, layout_engine)
  File "/usr/local/lib/python3.9/site-packages/PIL/ImageFont.py", line 173, in __init__
    freetype_version = parse_version(features.version_module("freetype2"))
  File "/usr/local/lib/python3.9/site-packages/packaging/version.py", line 49, in parse
    return Version(version)
  File "/usr/local/lib/python3.9/site-packages/packaging/version.py", line 264, in __init__
    match = self._regex.search(version)
TypeError: expected string or bytes-like object
```

Config:
```
    texts:
      - text: "Bedroom"
        x: 20
        y: 40
        color: [0, 0, 0]
        font: "NotoSans-BoldItalic.ttf"
        font_size: 10
```

Version HA:
Версия | core-2021.7.3
Тип установки | Home Assistant OS
Среда разработки | false
Supervisor | true
Docker | true
Виртуальное окружение | false
Версия Python | 3.9.5
Семейство операционных систем | Linux
Версия операционной системы | 5.10.17-v8
Архитектура ЦП | aarch64
Часовой пояс | Europe/Moscow

I try install font "apk add font-noto", then fc-list show font NotoSans-BoldItalic.ttf
If I comment #font: "NotoSans-BoldItalic.ttf", map work.

How use custom font in my version HA?

**Comment by PiotrMachowski** [comment]
Duplicate of #75 

Where have you executed `apk add font-noto` and `fc-list` commands?

**Comment by vashilov** [comment]
> Where have you executed `apk add font-noto` and `fc-list` commands?

Terminal addon

![Снимок экрана от 2021-07-27 11-17-48](https://user-images.githubusercontent.com/71368470/127120907-5e0bb9bc-6902-40d9-ad99-cfd6dcf31449.png)

**Comment by PiotrMachowski** [comment]
So you have probably added it in HA OS, to make it work you would have to do it in Home Assistant container (`docker exec -ti homeassistant`)

**Comment by vashilov** [comment]
I have only HA OS, no docker.
Am I right?

![Снимок экрана от 2021-07-27 11-26-13](https://user-images.githubusercontent.com/71368470/127122027-5f47a536-0b28-4471-b5e9-84c8f10aa3f9.png)

Are the fonts working now?

**Comment by PiotrMachowski** [comment]
HA OS runs Home Assistant in docker.

Do you use [this addon](https://github.com/hassio-addons/addon-ssh)?

**Comment by vashilov** [comment]
I use https://github.com/home-assistant/addons/tree/master/ssh
I am install HAOS on Raspberry from https://github.com/home-assistant/operating-system/releases/download/6.1/haos_rpi4-64-6.1.img.xz
It use docker?

**Comment by PiotrMachowski** [comment]
Yup, it internally uses docker (HA and all add-ons run in docker). Try using [this add-on](https://github.com/hassio-addons/addon-ssh), it does have access to docker command

**Comment by vashilov** [comment]
I install addon, and install apk add font-noto in docker.
Fonts no work.

I try
    texts:
      - text: "Bedroom"
        x: 20
        y: 40
        color: [0, 0, 0]
        font: "/usr/share/fonts/noto/NotoSans-BoldItalic.ttf"
and
    texts:
      - text: "Bedroom"
        x: 20
        y: 40
        color: [0, 0, 0]
        font: "NotoSans-BoldItalic.ttf"

**Comment by PiotrMachowski** [comment]
There seems that there is still some stuff missing, I'm not sure how to address it :/

**Comment by PiotrMachowski** [comment]
can you try adding `font_size`?
```
texts:
  - text: "Bedroom"
    x: 20
    y: 40
    color: [0, 0, 0]
    font: "NotoSans-BoldItalic.ttf"
    font_size: 20
```

**Comment by vashilov** [comment]
I try it.
Without font or font_size addon work, but no change font.
If I add font and font_size, then error.

P.S. Do you speak Russian?

**Comment by PiotrMachowski** [comment]
If any of `font` and `font_size` is not provided then integration doesn't even try to draw texts.
Were texts drawn correctly after [this change](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/75#issuecomment-888288299)?

Nope, just English and Polish

**Comment by vashilov** [comment]
Texts no drawn correctly.
I just don't understand why the changes aren't applied. Only after rebooting the entire HA?

**Comment by PiotrMachowski** [comment]
Changes can be applied by:
- HA reboot
- calling `xiaomi_cloud_map_extractor.reload` service
- Clicking "Reload xiaomi_cloud_map_extractor" in Server Control

**Comment by vashilov** [comment]
After edit config, "Reload xiaomi_cloud_map_extractor" ok.
If I edit .py files "Reload xiaomi_cloud_map_extractor" no work (i try do epty  /config/custom_components/xiaomi_cloud_map_extractor/common/image_handler.py, addon work).
Only if I reboot HA it  crash addon.

**Comment by PiotrMachowski** [comment]
Python files are loaded only during HA start, so this is a correct behavior
